game/lib/game/world_objects/world_object.py

```diff
@@ -0,0 +1,30 @@
+import pygame
+
+class WorldObject:
+
+    __real_x = None
```

Comment:
Można tutaj dodać pole __object_type, wtedy jak dostaniecie listę obiektów od serwera to będziecie wiedzieć co to za obiekt i łatwo będzie rzutować.